app/styles/partials/utils.scss

```diff
@@ -1,3 +1,7 @@
 .x-scrollable {
   overflow-x: auto;
 }
+
+.selection.dropdown.no-min-width {
+  min-width: 0em;
```

Comment:
@niranjan94 @CosmicCoder96 
`ui-dropdown` has a min-width of 14em which overflows when wrapped around a fixed width element. In this case field.